### ISSUE TITLE
Include ivy dependencies in a sbt/scala project?

### ISSUE THREAD

**Issue by oschrenk** [lukehutch/fast-classpath-scanner#51]
At the moment I'm using a copy/paste approach to get the classpath entries. Swapping to your library would ease development, but I miss some dependencies from a sbt/scala project in the classpath.

Example:

```
object ClassPathTest extends App {

  def urlses(cl: ClassLoader): Array[java.net.URL] = cl match {
    case null => Array()
    case u: java.net.URLClassLoader => u.getURLs() ++ urlses(cl.getParent)
    case _ => urlses(cl.getParent)
  }

  val  urls = urlses(getClass.getClassLoader)
  println(urls.mkString("\n"))

  println("")
  println("----------------------------------------")
  println("")

  val items = new ClasspathFinder().getUniqueClasspathElements();

  println(items)
```

prints

```
file:/Users/oliver/Projects/personal/fast-classpath/target/scala-2.11/classes/
file:/Users/oliver/.ivy2/cache/org.scala-lang/scala-library/jars/scala-library-2.11.8.jar
file:/Users/oliver/.ivy2/cache/com.google.code.gson/gson/jars/gson-2.4.jar
file:/Users/oliver/.ivy2/cache/io.github.lukehutch/fast-classpath-scanner/jars/fast-classpath-scanner-1.9.18.jar
file:/Users/oliver/.ivy2/cache/org.scala-lang/scala-library/jars/scala-library-2.11.8.jar
file:/Users/oliver/.ivy2/cache/org.scala-lang/scala-compiler/jars/scala-compiler-2.11.8.jar
file:/Users/oliver/.ivy2/cache/org.scala-lang/scala-reflect/jars/scala-reflect-2.11.8.jar
file:/Users/oliver/.ivy2/cache/org.scala-lang.modules/scala-xml_2.11/bundles/scala-xml_2.11-1.0.4.jar
file:/Users/oliver/.ivy2/cache/org.scala-lang.modules/scala-parser-combinators_2.11/bundles/scala-parser-combinators_2.11-1.0.4.jar
file:/Users/oliver/.ivy2/cache/jline/jline/jars/jline-2.12.1.jar
file:/usr/local/Cellar/sbt/0.13.11/libexec/sbt-launch.jar
file:/Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre/lib/ext/cldrdata.jar
file:/Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre/lib/ext/dnsns.jar
file:/Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre/lib/ext/jfxrt.jar
file:/Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre/lib/ext/localedata.jar
file:/Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre/lib/ext/nashorn.jar
file:/Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre/lib/ext/sunec.jar
file:/Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre/lib/ext/sunjce_provider.jar
file:/Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre/lib/ext/sunpkcs11.jar
file:/Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre/lib/ext/zipfs.jar
file:/System/Library/Java/Extensions/MRJToolkit.jar

----------------------------------------

[/usr/local/Cellar/sbt/0.13.11/libexec/sbt-launch.jar]
```

Is there a way to also print the ivy dependencies?

**Comment by lukehutch** [comment]
Hi Oliver,

Thanks for this report. Are you using the latest released version in Maven, or the head version in git? I pushed a possible fix for this 3 weeks ago, but haven't pushed out a new release yet, as I'm working on some other changes too.

I just pushed an additional possible fix to git, based on your code. Can you please test, and let me know if the head version is working for you now? Thanks!

**Comment by oschrenk** [comment]
I was using 1.9.18 from Maven Central.

Just tried HEAD (although I don't see your changes you mentioned available on Github) and I think the changes you made for #44 also apply to me. Works fine for me now.

Would be awesome if you could cut a new release as I was looking into this because of

https://github.com/MachinePublishers/jBrowserDriver/issues/41

An update to your lib might make jBrowserDriver more usable from Scala projects

**Comment by lukehutch** [comment]
Hi, thanks for testing. You're right, egit had a commit that was not properly pushed to master. Can you please test again with the changes I just pushed?

I'll be pushing out a new release in the next day or two to Maven Central, once I finish the other refactoring work that I'm currently undertaking.

**Comment by oschrenk** [comment]
Issue was already fixed on HEAD, still works with your latest changes on my small example. I'm also very happy to report that https://github.com/MachinePublishers/jBrowserDriver/issues/41 is fixed for me now.

**Comment by lukehutch** [comment]
Great, thank you for verifying!

**Comment by lukehutch** [comment]
Fixed in 1.9.19.
